tap/passive_tapper.go

```diff
@@ -96,6 +96,8 @@ var ownIps []string             // global
 var hostMode bool               // global
 var extensions []*api.Extension // global
 
+var filteringOptions *api.TrafficFilteringOptions
```

Comment:
add //global comment (to be like others..)

Reply:
@gadotroee added with https://github.com/up9inc/mizu/commit/73ed5b9a7776cfe87f77c9fe86159e79e57f0339

Reply:
👍